Index: version_1/test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- version_1/test.py	(date 1569016318000)
+++ version_1/test.py	(date 1569016676379)
@@ -17,6 +17,7 @@
     def test_01_load_page(self):
         driver = self.driver
         assert "Поліклініка без черг" in driver.title
+        print(driver.title)
 
     def test_02_1_enter_submit_without_login(self):
         driver = self.driver
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(date 1569016318000)
+++ .gitignore	(date 1569016653823)
@@ -113,6 +113,7 @@
 # Rope project settings
 .ropeproject
 
+
 # mkdocs documentation
 /site
 
